email_validator/deliverability.py

```diff
@@ -1,4 +1,5 @@
-from typing import Any, Dict, Optional
+from typing import Any, List, Optional, Tuple, TypedDict
+from typing_extensions import NotRequired
```

Comment:
typing_extensions is available for tests because it's in the test_requirements.txt file but it's not in the main requirements list.

Reply:
Done (added to the main requirements list).

Reply:
Can we use `total=False` on the TypedDict definition so that NotRequired isn't necessary, and then the new dependency doesn't need to be added?

Reply:
Yep, done.